docs/quickstart.md

```diff
@@ -113,7 +108,6 @@ Once installed, create a new environment and open it (here we use Python 3.7 but
 touch Pipfile && pipenv --python 3.7 && pipenv shell
 ```
 
-For more guidance on setting up a development environment check out <a href="../development-setup">this guide</a>.
```

Comment:
On doing “aea fetch fetchai/my_first_aea:0.25.0” below, strictly following the guide, I got this error:
```
	…
	f"Registry path not provided and `{DEFAULT_REGISTRY_NAME}` not found in current ({self.cwd}) and parent directory."
	ValueError: Registry path not provided and `packages` not found in current (.) and parent directory.
```

So the empty `packages` folder and the `.env` file, outlined in development-setup is required.

From the outset in Preliminary, we should direct users to follow the “development setup” guide which describes both the template repo option and the manual option, and we can remove the manual setup text and the template repo bit under preliminary.

Reply:
Not strictly needed, we just would need to use `--remote` flag in fetch/add. But it's probably better to just ask them to create an empty `packages` dir or fetch the repo. I would not make fetching the repo as a requirement.